src/main/java/org/tron/common/entity/NodeInfo.java

```diff
@@ -1,7 +1,9 @@
 package org.tron.common.entity;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class NodeInfo {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a constructor to the class, or provide default values. ([squid:S1258](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1258))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-vKlHK86Hco_1fM&open=AWalU-vKlHK86Hco_1fM)

[ISSUE_KEY]: AWalU-vKlHK86Hco_1fM